src/hubbleds/pages/01-spectra-&-velocity/component_state.py

```diff
@@ -114,6 +115,22 @@ class ComponentState(BaseComponentState, BaseState):
     velocity_reflection_state: VelocityReflection = VelocityReflection()
     reflection_complete: bool = False
     show_dop_cal4_values: bool = False
+    
```

Comment:
My one suggestion would be that we move these two computed fields into the `BaseComponentState` class so that we don't have to maintain this five times.

Reply:
Yeah, I thought this would be a good thing to have in the base component state as well. I will add that